Tools/MGCB/BuildContent.cs

```diff
@@ -261,149 +299,201 @@ public void Build(out int successCount, out int errorCount)
 
             // If the target changed in any way then we need to force
             // a fuull rebuild even under incremental builds.
+            // jcf: what? why?
```

Comment:
`previousContent` is what the intermediate data says was previously built.  If you decided to change what platform you are building for, but didn't change the folder it build to...  then the output and intermediate folders would have content for a different platform in it.  This code is to force a clean if you changed these settings and then wrote to the same folder.